src/libraries/Microsoft.PowerFx.Connectors/Tabular/Services/CdpTabularService.cs

```diff
@@ -24,6 +24,10 @@ public class CdpTabularService : TabularService
 
         public string TableName { get; protected set; }
 
+        public override bool IsDelegable => TableCapabilities?.IsDelegable ?? false;
+
+        internal ServiceCapabilities TableCapabilities { get; private set; }
```

Comment:
> ServiceCapabilities TableCapabilities  [](http://example.com/codeflow?start=16&length=39)

Should this be on base class?

Reply:
This object is really part of CDP protocol
Not that we shouldn't have capabilities at TabularService level but those should be abstracted which isn't the case here.